### PR TITLE
gnatformat and lal_refactor 25.0.0

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

**Comment by reznikmm** [comment]
this works for compiling ALS 25.0.0 fine.

File: index/gn/gnatformat/gnatformat-25.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name = "gnatformat"
+description = "Opinionated code formatter for the Ada language"
+version = "25.0.0"
+
+authors = ["AdaCore"]
+maintainers = ["Fabien Chouteau <chouteau@adacore.com>"]
+maintainers-logins = ["Fabien-Chouteau"]
+licenses = "MIT OR Apache-2.0 WITH LLVM-exception"
+website = ""
+tags = ["formating", "ada", "tool"]
```
Comment (mosteo) [inline index/gn/gnatformat/gnatformat-25.0.0.toml]: typo: formatting

File: index/li/libgnatformat/libgnatformat-25.0.0.toml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name = "libgnatformat"
+description = "opinionated code format library for the Ada language"
+version = "25.0.0"
+
+authors = ["AdaCore"]
+maintainers = ["Fabien Chouteau <chouteau@adacore.com>"]
+maintainers-logins = ["Fabien-Chouteau"]
+licenses = "MIT OR Apache-2.0 WITH LLVM-exception"
+website = ""
+tags = ["formating", "ada", "lib"]
```
Comment (mosteo) [inline index/li/libgnatformat/libgnatformat-25.0.0.toml]: typo: formatting

File: index/li/liblal_refactor/liblal_refactor-24.0.0.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name = "liblal_refactor"
-description = "Source code refactoring utilities for the Ada "
+description = "Source code refactoring utilities for the Ada"
```
Comment (mosteo) [inline index/li/liblal_refactor/liblal_refactor-24.0.0.toml]: I presume the intention was "the Ada language"?

File: index/la/lal_refactor/lal_refactor-25.0.0.toml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+name = "lal_refactor"
+description = "Source code refactoring utilities for the Ada"
```
Comment (mosteo) [inline index/la/lal_refactor/lal_refactor-25.0.0.toml]: "the Ada [language]" I guess

File: index/li/liblal_refactor/liblal_refactor-25.0.0.toml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+name = "liblal_refactor"
+description = "Source code refactoring library for the Ada"
```
Comment (mosteo) [inline index/li/liblal_refactor/liblal_refactor-25.0.0.toml]: same about "Ada [language]"

**Review by mosteo — CHANGES_REQUESTED** [review]
Just a few minor typos

**Review by mosteo — APPROVED** [review]

**Pull request merged**